src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditorViewModel.java

```diff
@@ -0,0 +1,55 @@
+package org.jabref.gui.fieldeditors;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.util.StringConverter;
+
+import org.jabref.gui.util.BindingsHelper;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.EntryLinkList;
+import org.jabref.model.entry.ParsedEntryLink;
+
+public class LinkedEntriesEditorViewModel extends AbstractEditorViewModel {
+
+    private final BibDatabaseContext databaseContext;
+    private final ListProperty<ParsedEntryLink> linkedEntries;
+
+    public LinkedEntriesEditorViewModel(BibDatabaseContext databaseContext) {
+        this.databaseContext = databaseContext;
+        linkedEntries = new SimpleListProperty<>(FXCollections.observableArrayList());
+        BindingsHelper.bindContentBidirectional(
+                linkedEntries,
+                text,
+                EntryLinkList::serialize,
+                newText -> EntryLinkList.parse(newText, databaseContext.getDatabase()));
+    }
+
+    public ListProperty<ParsedEntryLink> linkedEntriesProperty() {
+        return linkedEntries;
+    }
+
+    public StringConverter<ParsedEntryLink> getStringConverter() {
+        return new StringConverter<ParsedEntryLink>() {
+            @Override
+            public String toString(ParsedEntryLink linkedEntry) {
+                if (linkedEntry == null) {
+                    return "";
+                }
+                return linkedEntry.getKey();
+            }
+
+            @Override
+            public ParsedEntryLink fromString(String key) {
+                return databaseContext.getDatabase().getEntryByKey(key).map(ParsedEntryLink::new).orElse(null);
+            }
+        };
+    }
+
+    public void jumpToEntry(ParsedEntryLink parsedEntryLink) {
+        // TODO: Implement jump to entry
```

Comment:
I see. It would be better to keep it in the language file. Thus, I would say it is ok to keep the code, but please extend the explanation, so that we know that this is not just the legacy stuff. Be verbose (no-problem with a multi-line comment) and explain the rationale and why it's problematic to re-add the code, like in you comment here.